internal/command/delta.go

```diff
@@ -24,10 +23,13 @@ import (
 
 func init() {
 	deltaCmd.Flags().StringVarP(&scoreFile, "file", "f", scoreFileDefault, "Source SCORE file")
+	deltaCmd.MarkFlagRequired("file")
```

Comment:
Would this still work if `-f` (or `--file`) is omitted? Default value should apply.

Reply:
aha yes woops there's a default. will remove that